src/App.js

```diff
@@ -34,6 +34,10 @@ type AppContext = {
     bluetoothApiIsAvailable: boolean
 };
 
+type AppProps = {
+    movementDelay: number
```

Comment:
As we discussed briefly this morning during planning, I'd suggest moving the `movementDelay` variable to `AppSettings`, an instance of which is stored inside `AppState` under the `settings` key. This is where we are currently storing user configurable settings that we might want to persist in the future. Let's also add units to the variable name: `movementDelayMs`